iocore/net/UnixNetAccept.cc

```diff
@@ -225,7 +233,7 @@ NetAccept::init_accept_per_thread()
     }
 
     a->mutex = get_NetHandler(t)->mutex;
-    t->schedule_every(a, period, opt.etype);
```

Comment:
And the opt.etype didn't do anything anyway?

Reply:
The 3rd parameter of EThread::schedule_every() is callback_event. It can be ignored because NetAccept::acceptFastEvent does not check the value of event.